src/settings/toml/target.rs

```diff
@@ -12,6 +12,7 @@ pub struct Target {
     pub account_id: String,
     pub kv_namespaces: Vec<KvNamespace>,
     pub name: String,
+    pub main: String,
```

Comment:
nit: it would be nice if this was an `Option<String>`, and you used serde's `string_empty_as_none` for the entry in `manifest.rs`